mentors/java/m.json

```diff
@@ -63,6 +63,14 @@
   "avatar_url": null,
   "bio": "Mostly a Java developer, interested in solving problems and always looking for ways to learn new things. I'm also into cross-country skiing!"
 },
+{
+  "github_username": "MindLabor",
+  "name": "Samuel Braun",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "My first programming language was Java. At the beginning it was quite challenging to write some code that does actually make sense. After a few days however, it became more and more easier and fun to mess around with."
```

Comment:
Might want `easy` instead of `easier`.

Reply:
Yeah, 'easy' makes more sense. I'm from Germany and English isn't my native language, so....

Reply:
....or you could remove `more and more`...

Reply:
I don't know. It's up to you. You're the native speaker. I am sure you will choose wisely ;)

Reply:
Either suggestion is fine.  However, I cannot make the change.  I can only approve and merge.